docs/interactive-vm.md

```diff
@@ -0,0 +1,17 @@
+# Running Interactive VMs with disko
+
+disko now exports it's own flavor of interactive VMs (similiar to config.system.build.vm).
```

Comment:
Since we normal build-vm stuff is broken, should we maybe mention this here?

Reply:
whats broken?

Reply:
See the details here: https://github.com/nix-community/disko/issues/668